steps/localazy/1.0.1/step.yml

```diff
@@ -0,0 +1,96 @@
+title: Localazy CLI
+summary: Runs Localazy CLI and download updated translations
+description: "This step runs [Localazy CLI](https://localazy.com/docs/cli) and download
+  updated translations.\n[Localazy](https://localazy.com) is a translation management
+  platform built by developers for developers \nto localize apps in the modern way. "
+website: https://localazy.com
+source_code_url: https://github.com/localazy/bitrise-step-localazy
+support_url: https://github.com/localazy/bitrise-step-localazy/issues
+published_at: 2020-11-26T14:12:21.200962895+01:00
+source:
+  git: https://github.com/localazy/bitrise-step-localazy.git
+  commit: 8190f3c85a295128b4a98fa72da1ca2f78fb29a0
+project_type_tags:
+- ios
```

Comment:
If all projects types are supported then list may not be specified.

Reply:
Should we remove it and create a new PR? I will remove it for subsequent releases.

Reply:
I submitted another PR with this issue fixed.